hack/terraform/config.tf

```diff
@@ -14,7 +14,16 @@ terraform {
     region = "ap-southeast-1"
   }
 }
-
 provider "aws" {
   region = "ap-southeast-1"
 }
+
+variable "s3_bucket_names" {
+  type = list(string)
+  default = ["download.devstream.io", "just-for-test"]
+}
+
+resource "aws_s3_bucket" "devstream_buckets" {
+  count         = length(var.s3_bucket_names)
```

Comment:
Do not use "count". Please use "for_each" https://www.terraform.io/language/meta-arguments/for_each.

Reply:
Count has been replaced by for_each.